lib/pingen/response.rb

```diff
@@ -6,6 +6,8 @@ class Response
 
     def self.from(response)
       new(JSON.parse(response.body, symbolize_names: true), response.code.to_i)
+    rescue JSON::ParserError
```

Comment:
did it happen?

Reply:
yes, for the `pdf` method, it obviously doesn't return JSON but a PDF file.